.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -1253,6 +1373,18 @@ iam_CreateRole:
   synopsis: create an &IAM; role.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreateRoleExample
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.CreateRoleAsync
```

Comment:
These two examples are almost identical. Include only one--I'd pick the second because it's the tested one, but it's your call.

Reply:
Removed the first.